### PR TITLE
fix(workflow): return case status on complete / skip work-item mutation

### PR DESCRIPTION
**Pull request by luytena**

### CODE REVIEW

File: packages/form/tests/integration/components/cf-field/input/action-button-test.js [hunk 1]
```diff
@@ -16,9 +16,14 @@ module(
...
 
+      const workItem = this.server.create("work-item");
+      workItem.update({
+        case: this.server.create("case"),
+      });
+
       this.field = {
         document: {
-          workItemUuid: this.server.create("work-item").id,
+          workItemUuid: workItem.id,
```
Comment (anehx) [inline packages/form/tests/integration/components/cf-field/input/action-button-test.js:26]:
```suggestion
          workItemUuid: this.server.create("work-item", { case: this.server.create("case") }).id,
```

Comment (luytena) [reply packages/form/tests/integration/components/cf-field/input/action-button-test.js]: Done :+1:

**Review by anehx — APPROVED** [review]

**Pull request merged**